monai/transforms/io/array.py

```diff
@@ -107,9 +107,12 @@ def __init__(
             reader: reader to load image file and meta data
 
                 - if `reader` is None, a default set of `SUPPORTED_READERS` will be used.
-                - if `reader` is a string, the corresponding item in `SUPPORTED_READERS` will be used,
-                  and a reader instance will be constructed with the `*args` and `**kwargs` parameters.
-                  the supported reader names are: "nibabelreader", "pilreader", "itkreader", "numpyreader".
+                - if `reader` is a string, it will follow the order to get target reader class:
+                  1. search all the reader classes in `monai.data` module.
```

Comment:
perhaps we don't need to explain the implementation details in the docstring? I wrote this docstring and I think it's concise https://github.com/Project-MONAI/MONAI/blob/d8121d235178c8a8b123a71f4cd53b23e79dac2d/monai/transforms/io/array.py#L306-L307